markdown/events/2022/bytesize-survey_results.md

```diff
@@ -25,3 +25,87 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[11:23](https://youtu.be/DYdPNPVa4wct=683)
+Last year we asked about Kubernetes, a bit of a hot topic for those of you who know about it. Lots of people, a small number of people, I think it was about 8% last year, who are already using Kubernetes. Lots of people saying you're planning to use Kubernetes in the future. We're curious to see if anything changes in the year. It hasn't. There's pretty much exactly the same number of people saying that they are actively using Kubernetes today across the various different Kubernetes solutions. But again, lots of people saying that they're interested in it. Moving forward, we'll see if anything changes there in the future.
+
+[12:07](https://youtu.be/DYdPNPVa4wct=727)
+We asked a bit about the different traits, the reasons that you're use Nextflow, what things do you find important when you're choosing which tool to use. And the winner of this category was definitely documentation. I'm totally with you on this one. I've got a soft spot for trying to put together documentation for tools. Documentation and performance are the two outstanding categories here. And when we asked you, OK, within documentation and learning materials, what do you use the most, what's most useful? The official documentation, the reference documentation came top and nf-core came a close second. That's great to see. Everyone is using the documentation that we've been building. And this is definitely a hot topic for us right now. We've got lots of room for improvement with documentation. That's good to see.
+
+[13:08](https://youtu.be/DYdPNPVa4wct=788)
+This survey went out just before the Nextflow slack went live. Everyone was still using the Nextflow gitter at that point. But we asked about the nf-core slack. And the vast majority of people who responded knew about and used the nf-core slack, which is great. Not only is the nf-core community building pipelines and standards, but it's also a big support channel. And lots of people are also feeding back into the community. Nearly 30 percent of people are contributing back to these nf-core community pipelines. It's really good to see. It's a two way street there.
+
+[13:46](https://youtu.be/DYdPNPVa4wct=826)
+Apparently no one really wants a graphical interface for their workflow manager. It's fair enough. But you can see documentation really stands out as being very important to lots of people. Yes. And then also we're thinking about integrations and this is more tooling. Again, it's quite detailed here. But a lot of you want to be able to optimize computational resources, which makes sense. Unit tests was a popular category here. This is great. This is real fodder here for feeding into the Nextflow development process, and nf-core, to really prioritize which topics need to be tackled.
```

Comment:
```suggestion
Apparently no one really wants a graphical interface for their workflow manager. It's fair enough. But you can see documentation really stands out as being very important to lots of people. Yes. And then also we're thinking about integrations and this is more tooling. Again, it's quite detailed here. But a lot of you want to be able to optimize computational resources, which makes sense. Unit tests was a popular category here. This is great. This is can a real fodder here for feeding into the Nextflow development process, and nf-core, to really prioritize which topics need to be tackled.
```

Reply:
Hi @laribritto, some changes I made are to make it more readable and understandable. This seems one of them.